File: php/phpsearch/src/phpsearch/Color.php
<?php

declare(strict_types=1);

namespace phpsearch;

/**
 * Enum Color
 */
enum Color: string
{
    case Grey = "\033[30m";
    case Red = "\033[31m";
    case Green = "\033[32m";
    case Yellow = "\033[33m";
    case Blue = "\033[34m";
    case Reset = "\033[0m";
}

File: haskell/hssearch/src/HsSearch/Color.hs
module HsSearch.Color
  (
      reset
    , black
    , red
    , green
    , yellow
    , blue
    , purple
    , magenta
    , cyan
    , white
  ) where


reset :: String
reset = "\x1b[0m"

black :: String
black = "\x1b[30m"

red :: String
red = "\x1b[31m"

green :: String
green = "\x1b[32m"

yellow :: String
yellow = "\x1b[33m"

blue :: String
blue = "\x1b[34m"

purple :: String
purple = "\x1b[35m"

magenta :: String
magenta = purple

cyan :: String
cyan = "\x1b[36m"

white :: String
white = "\x1b[37m"
